wger/utils/language.py

```diff
@@ -52,7 +52,7 @@ def load_language(language_code=None):
         language = Language.objects.get(short_name=used_language)
     except ObjectDoesNotExist:
         # No luck, load english as our fall-back language
-        language = Language.objects.get(short_name="en")
+        language = Language.objects.get_or_create(short_name="en")
```

Comment:
change this line back to `language = Language.objects.get(short_name="en")`

Reply:
I will do that on my next commit